src/connection-manager/index.ts

```diff
@@ -20,7 +19,6 @@ async function start(): Promise<void> {
   }
   await Promise.all(promises)
   await startConnections()
-  await addAmendmentsDataFromJSON()
```

Comment:
Wasnt this just added in the previous PR?  Why was it moved again?

Reply:
The one added on the previous PR was `fetchAmendmentInfo` which fetches amendments info from the 2 links. This one fetches amendments data from JSON, and I had it ran on `setupTables` now since it only runs once when we redeploy